functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/project/ProjectDashboardSpec.groovy

```diff
@@ -0,0 +1,55 @@
+package org.rundeck.tests.functional.selenium.project
+
+import org.openqa.selenium.support.ui.WebDriverWait
+import org.rundeck.util.annotations.SeleniumCoreTest
+import org.rundeck.util.common.jobs.JobUtils
+import org.rundeck.util.container.SeleniumBase
+import org.rundeck.util.gui.pages.login.LoginPage
+import org.rundeck.util.gui.pages.project.DashboardPage
+import spock.lang.Shared
+
+import java.time.Duration
+
+@SeleniumCoreTest
+class ProjectDashboardSpec extends SeleniumBase {
+    @Shared
+    WebDriverWait wait
+
+    @Shared
+    String projectName = "dashboard-test-project"
+
+    @Shared
+    String jobId
```

Comment:
this is not needed here, the definition can be done inside the "setupSpec" method